ansible/roles/ocp4-workload-integreatly/defaults/main.yml

```diff
@@ -24,4 +24,21 @@ silent: False
 ocp4_workload_integreatly_defaults:
     variable_1: "value 1"
     variable_2: "value 2"
-    variable_secret: ""
\ No newline at end of file
+    variable_secret: ""
+    action: install
+    namespace: redhat-rhmi-3scale
+    seed_secret_name: system-seed
+    tmp_dir: /tmp
+    # default tenant details
+    tenant_name: admin-tenant
+    tenant_admin_user: admin
+    tenant_admin_email: admin@example.com
+    tenant_admin_password: Password1
+    tenant_detail_secret_name: admin-tenant
+    tenant_detail_secret_namespace: "redhat-rhmi-3scale"
+    tenant_id_format: evals%02d-tenant
+    users_name_format: evals%02d
+    user_count : 50
```

Comment:
Since we'll be using the auth module for setting up htpwd users, what will happen if these variables change? Should we be setting them dynamically?

Reply:
could probably use the vars @ciaranRoche is using https://github.com/redhat-cop/agnosticd/pull/1644/files#diff-84a91b3688b214d38eeb8f05066f999dR10-R11